commands/team_e2e_test.go

```diff
@@ -51,3 +52,48 @@ func (s *MmctlE2ETestSuite) TestRenameTeamCmdF() {
 		s.Equal("Cannot rename team '"+s.th.BasicTeam.Name+"', error : : You do not have the appropriate permissions., ", err.Error())
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestModifyTeamsCmdF() {
+	s.SetupTestHelper().InitBasic()
+	s.RunForSystemAdminAndLocal("system & local accounts can set a team to private", func(c client.Client) {
+		printer.Clean()
+		teamID := s.th.BasicTeam.Id
+		cmd := &cobra.Command{}
+		cmd.Flags().Bool("private", true, "")
+		err := modifyTeamsCmdF(c, cmd, []string{teamID})
+		s.Require().NoError(err)
```

Comment:
Could you please add an assertion for the returned team as well? Same suggestion applies for the other test cases

Reply:
Hi @mgdelacroix where do I obtain the returned team?

Reply:
Hey! I think I did it.

Thanks a ton for the suggestion!